src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    private static final int MAX_NUMBER_OF_BALLS = 100;
+
+    @Override
+    public String getRandomColor() {
+        return Colors.values()[new Random().nextInt(Colors.values().length)].toString();
+    }
+
+    public String getRandomBall() {
```

Comment:
getRandomBall() should return a Ball object instead of a String. You should create a new Ball instance with the random color and random number, and return it. Example: return new Ball(colorSupplier.getRandomColor(), RANDOM.nextInt(MAX_BALL_NUMBER));